opsdroid/helper.py

```diff
@@ -19,7 +23,12 @@ def get_opsdroid():
 
 
 def del_rw(action, name, exc):
-    """Error handler for removing read only files."""
+    """This method return an error handler for removing.
+        This method allow users to read only files.
+
+        Raises:
+            KeyError : Raises an exception
```

Comment:
The function doesn't raise an exception itself but `os.remove` does, it should be useful to mention this more explicitly, also the exception raise would be "OsError" the message can be something like: Raised if file to be removed is a directory.